### PR TITLE
Add past proposals and other grant info to Funding page

### LINKED ISSUE #76: Add list of Jupyter accessibility-focused funding to Funding page
On the [documentation site for this repo](https://jupyter-accessibility.readthedocs.io/en/restructure/README.html), there is a [Funding](https://jupyter-accessibility.readthedocs.io/en/restructure/docs/funding/README.html) page. There is very little information here, so I propose we add a list of existing funding (and maybe one day past funding, when we get there). 

This could also be a good place to link resources related to funding, like [past proposal documents](m), [roadmaps](https://github.com/jupyter/accessibility/blob/master/docs/source/meeting-minutes/czi-grant-updates/czi-grant-roadmap.md) so on. (Both of which are, at the moment, resources only findable by navigating through a series of directories in the repo. Which isn't particularly discoverable.)

I'm happy to make a PR for this (and addressing the [roadmap to-dos that didn't happen prior to merge](https://github.com/jupyter/accessibility/pull/64#issue-1087323852)) as long as I know what's wanted.

Thoughts? Concerns?

### PR DESCRIPTION
**Pull request by isabela-pf**
This PR
- Adds a section for past proposals to the [Funding page](https://jupyter-accessibility.readthedocs.io/en/restructure/docs/funding/README.html)
- Moves the CZI EOSS grant roadmap to an appropriate directory and links to in from Funding

Fixes #76 

It is in draft mode because I need to make changes not completed before #64 was merged. They include:
- [x] Create a file to listed completed roadmap items (also good so people can double check that changes are working)
- [x] Check if I need to update the docs index (or if this is grouped in fine with the meeting notes)
- [x] Make sure the intro has proper context for this repository. Maybe link to the proposal or other issues on this repo that are relevant.
- [x] Link to this from the readme

Even though there's more to do, I'm open to review now. Thank you!

### CODE REVIEW

**Comment by isabela-pf** [comment]
I'm marking this as ready for review because I've completed all the to dos! Feedback would be appreciated.

File: docs/funding/czi-grant-completed.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Jupyter Accessibility Roadmap - Completed Items
+
+Created on: April 1, 2022
```
Comment (gabalafou) [inline docs/funding/czi-grant-completed.md:3]: April Fools?

**Comment by isabela-pf** [comment]
Since I've gotten pretty neutral verbal feedback on this and we can always iterate, I'm going to merge this PR. Still let me know if you have any feedback and I can fix it in a follow up PR!

File: docs/funding/czi-grant-completed.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## In the works
+
+For items not yet completed, but next on the list, refer to the [Jupyter Accessibility Roadmap](czi-grant-roadmap.md)
+
+For the full scope of the two-year grant, all key items are listed on the [jupyter-a11y-mgmt repository's planning issues](https://github.com/orgs/Quansight-Labs/projects/5/views/1). 
+
+This roadmap's content is derived from
+- [The original grant proposal](https://github.com/jupyter/accessibility/blob/master/grant-applications/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf)
+- The [jupyter-a11y-mgmt repository's planning issues](https://github.com/orgs/Quansight-Labs/projects/5/views/1) 
+- The [JupyterLab accessibility project board](https://github.com/orgs/Quansight-Labs/projects/5/views/1)
```
Comment (gabalafou) [inline docs/funding/czi-grant-completed.md:20]: I think this is the wrong link (it's the same URL as the item above it)

File: docs/funding/czi-grant-completed.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Update the [JupyterLab accessibility project board](https://github.com/orgs/jupyterlab/projects/1)
+
+## In the works
+
+For items not yet completed, but next on the list, refer to the [Jupyter Accessibility Roadmap](czi-grant-roadmap.md)
+
+For the full scope of the two-year grant, all key items are listed on the [jupyter-a11y-mgmt repository's planning issues](https://github.com/orgs/Quansight-Labs/projects/5/views/1). 
+
+This roadmap's content is derived from
+- [The original grant proposal](https://github.com/jupyter/accessibility/blob/master/grant-applications/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf)
```
Comment (gabalafou) [inline docs/funding/czi-grant-completed.md:18]: I think this link is dead

**Comment by gabalafou** [comment]
Otherwise this looked good to me. Thanks for updating!

File: docs/funding/czi-grant-completed.md (hunk 3 above)
Comment (slel) [reply docs/funding/czi-grant-completed.md:18]: Correct link: https://github.com/jupyter/accessibility/tree/master/docs/funding/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf

File: docs/funding/czi-grant-completed.md (hunk 2 above)
Comment (slel) [reply docs/funding/czi-grant-completed.md:20]: Maybe one of these was meant:

- https://github.com/orgs/jupyterlab/projects/1
- https://github.com/jupyter/accessibility/discussions/32

**Pull request merged**